src/amr/multiphysics_integrator.h

```diff
@@ -18,14 +18,14 @@
 #include "amr/messengers/messenger.h"
 #include "amr/tagging/tagger.h"
 
-#include "solver/physical_models/hybrid_model.h"
-#include "solver/physical_models/mhd_model.h"
-#include "solver/physical_models/physical_model.h"
-#include "solver/solvers/solver.h"
-#include "solver/messenger_registration.h"
-#include "solver/level_initializer/level_initializer.h"
+#include "physical_models/hybrid_model.h"
```

Comment:
this compiles, but it probably shouldn't.

ie. there's a bunch of extra "-Isrc/$module" that are in cmake but I don't think should be

all includes should be from src